webcompat/form.py

```diff
@@ -111,12 +206,72 @@ class IssueForm(FlaskForm):
     submit_type = HiddenField()
 
 
-def get_form(form_data):
+class PrefixedRadioField(RadioField):
+    """Prefix radio field label with an image."""
+    def __init__(self, *args, **kwargs):
+        prefixed_choices = kwargs.pop('choices')
+        template = '<div class={cls}><img src={src}/></div> {text}'
+        choices = []
+
+        for slug, img, text in prefixed_choices:
+            cls = 'icon-container'
+            filename = 'img/svg/icons/{img}'.format(img=img)
+            src = url_for('static', filename=filename)
+            label = Markup(template.format(src=src, cls=cls, text=text))
+            choice = (slug, label)
+            choices.append(choice)
+
+        kwargs['choices'] = choices
+        super(PrefixedRadioField, self).__init__(*args, **kwargs)
+
+
+class IssueFormRedesigned(IssueForm):
+    """Re-designed version of IssueForm:
```

Comment:
nit: docstring doesn't follow our standard. #tofixlater

Though it's probably our fault to not enforce it through circleci check.